src/vm/marshalnative.cpp

```diff
@@ -610,21 +594,12 @@ FCIMPL1(LPVOID, MarshalNative::GCHandleInternalGet, OBJECTHANDLE handle)
 FCIMPLEND
 
 // Update the object referenced by a GC handle.
-FCIMPL3(VOID, MarshalNative::GCHandleInternalSet, OBJECTHANDLE handle, Object *obj, CLR_BOOL isPinned)
+FCIMPL2(VOID, MarshalNative::GCHandleInternalSet, OBJECTHANDLE handle, Object *obj)
 {
     FCALL_CONTRACT;
 
     OBJECTREF objRef(obj);
-    HELPER_METHOD_FRAME_BEGIN_1(objRef);
-    
-    if (isPinned)
-    {
-        ValidatePinnedObject(objRef);
-    }
-
-    // Update the stored object reference.
     StoreObjectInHandle(handle, objRef);
-    HELPER_METHOD_FRAME_END();
```

Comment:
@jkotas, did I correctly understand that I can remove the HELPER_METHOD_FRAME* stuff from here?  It doesn't look like StoreObjectInHandle can throw, but I may have missed it.

Reply:
👍